app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,15 @@
+require './config/environment'
+
+class TweetsController < ApplicationController
+
```

Comment:
Looks like the rest of the routes related to tweets can be moved from the application_controller to this tweets_controller.